src/styles/index.scss

```diff
@@ -2,2 +1,69 @@
+@import 'variables';
+
+html {
+  font-family: Arial, sans-serif;
```

Comment:
Be sure to follow the font-family fallback system correctly. The [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules) suggest using 'Helvetica Neue', Helvetica, Arial, sans-serif as the font-family fallback.